src/Storage/functions.php

```
@@ -0,0 +1,17 @@
+<?php
+
+namespace Google\Cloud\Storage;
+
+function registerStreamWrapper(string $protocol = null)
+{
+    $protocol = $protocol ?: 'gs';
+    if (!in_array($protocol, stream_get_wrappers())) {
+        stream_wrapper_register($protocol, 'Google\Cloud\Storage\StreamWrapper')
+            or die("Failed to register '$protocol://' protocol");
```

Can we throw an exception here instead? `\RuntimeException`?